src/System.Management.Automation/help/ManpageHelpInfo.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;      // for fxcop
+using System.Management.Automation.Help;    // for DefaultCommandHelpObjectBuilder
+
+namespace System.Management.Automation
+{
+    /// <summary>
+    /// Stores help information related to commands Manpage.
+    /// </summary>
+    internal class ManpageHelpInfo : BaseCommandHelpInfo
+    {
+        /// <summary>
+        /// Initializes a new instance of the ManpagesHelpInfo class.
+        /// </summary>
+        /// <remarks>
+        /// The constructor is private. The only way to create an
+        /// ManpageHelpInfo object is through static method <see cref="GetHelpInfo"/>
+        /// </remarks>
+        [SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
+        private ManpageHelpInfo(
+            ManpageInfo manpageInfo,
```

Comment:
This is only define on UNIX

Reply:
Fixed.  Few `#if UNIX` were missing.  Somehow I overlooked these errors because of the other tests failures. Sorry for that.

Regarding the remaining failures.  I don't have Mac, so I am not sure if the specific Mac failures are bugs or that tests should be modified because of the addition of the manpages info to the help output.

Regarding the Linux failures, I don't understand how the changes could affect the failing tests.  For example, [this failure](https://dev.azure.com/powershell/PowerShell/_build/results?buildId=78293&view=logs&j=0adc2cfe-3cfb-5a94-26e8-5272d0ebb7c1&t=3492351e-2f82-5045-f1f1-dc609a2d9afc&l=6070) - as there is no `helpfanct12` manpage, the changes should not have any effect on this test.